framework/core/migrations/2022_08_06_000000_change_access_tokens_last_activity_at_to_nullable.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Schema\Builder;
+
+return [
+    'up' => function (Builder $schema) {
+        $schema->table('access_tokens', function (Blueprint $table) {
+            $table->dateTime('last_activity_at')->nullable()->change();
```

Comment:
My thinking is, since now developer tokens *could* be created by the end-user now, they wouldn't be instantly used. So the last activity *should* be null to indicate that it was never used until it actually is.

Reply:
Sycho's thought here is the correct one. The last activity should always be null until it's actually used. That's a standard practice you'll find across almost all applications/services.